### DISCUSSION TITLE
Our general principles and monetization

### DISCUSSION THREAD

**Discussion by erupturatis** [NavigoLearn/Design#20] [General]
First of all, we don't want to monetize the roadmaps in any way. The most important thing for us is to keep the community vibe and have an "ideologically pure" product that will focus on the best experience and most value given for the user. In our case, we base our product on the satisfaction someone gets from contributing on open-source towards something everyone believes in and the allegiance to a community and a greater purpose.

Monetizing roadmaps will almost certainly lead to bad user experience and partly destroy the ideological foundation we try to build ( like placing ads on the roadmaps or restricting some features ). Some mitigation to this would come in part from sharing the money made from a roadmap with the creator (the youtube model ) but that comes with its own set of problems. For example it might lead to self-interest hurting the general purpose of creating roadmaps as good as possible ( someone does not want to share the money so they don't accept contribution and just copy them ? ) or as a contributor it will decrease your incentive to send a request because it can just be copied and instead make your own roadmap with that specific part , again hurting the initial goal of rallying people together to create the best roadmaps.

What we think at the moment is that the best model is something similar to github / vercel / redhat aka make a very good open source product and give it for free to everyone and then build some specific things around it that people can pay for but not having those will not affect the initial experience. We are not sure how that would translate in our case, but some example we thought of is allow users to monetize their guides ( see the guides feature ). That way it will not directly affect the roadmap, but you would be able to make some money off of it ( if you make good guides people will use them as links in their roadmaps therefore bringing you traffic ). This again might pose a problem when the owner of a roadmap will start creating their own guides and putting them in their own roadmap so we might think again about that too. Even if we have the ranking and like system we cannot count on them to account for this kind of behavior because "someone better will come and be more upvoted" because it would be extremely hard to unmount from the first places of ranking someone who has been entrenched for a long time.

TLDR: we don't have any plans yet for monetization, we think of a model similar to github/vercel/redhat that won't affect the goal of creating good roadmaps, we want your help and opinions
